src/renderer/store/exportIo/helpers.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * All input/output user actions states
+ * Import / Export progress states
+ */
+export interface ExportIO {
```

Comment:
For consistency, treat acronyms as regular words in camelcase e.g. `exportIo`

Reply:
Yeah good idea. Thanks. Fixed